src/styles/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  &__star-block {
+    display: flex;
+  }
+
+  &__reviews {
+    margin: 0;
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+  }
+
+  // .stars__star
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
The `background-image` URL should be relative to the `index.scss` file. Ensure the path is correct so that the image can be loaded properly.